.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,16 @@
 Descriptions of the changes in this PR:
 
-xxxxxx
+(PR description content here)...
 
 ---
 Be sure to do all of the following to help us incorporate your contribution
 quickly and easily:
 
 - [ ] Make sure the PR title is formatted like:
-    `<Issue #>: Description of pull request`
+    `<Issue # or JIRA #>: Description of pull request`
```

Comment:
@eolivelli does this address your comment? do we need to call out 'BOOKKEEPER-#' explicility?